assets/queries/terraform/gcp/service_account_with_improper_privileges/query.rego

```diff
@@ -21,6 +21,24 @@ CxPolicy[result] {
 	}
 }
 
+CxPolicy[result] {
+	resource := input.document[i].data.google_iam_policy[name]
+
+	tf_lib.check_member(resource.binding[x], "serviceAccount:")
+	has_improperly_privileges(resource.binding[x].role)
+
+	result := {
+		"documentId": input.document[i].id,
+		"resourceType": "google_iam_policy",
+		"resourceName": tf_lib.get_resource_name(resource, name),
+		"searchKey": sprintf("google_iam_policy[%s].binding[%s].role", [name, format_int(x, 10)]),
+		"issueType": "IncorrectValue",
+		"keyExpectedValue": sprintf("google_iam_policy[%s].binding[%s].role should not have admin, editor, owner, or write privileges for service account member", [name, format_int(x, 10)]),
+		"keyActualValue": sprintf("google_iam_policy[%s].binding[%s].role has admin, editor, owner, or write privilege for service account member", [name, format_int(x, 10)]),
+		"searchLine": common_lib.build_search_line(["resource", "google_iam_policy", name, "binding", x, "role"], []),
```

Comment:
```suggestion
		"searchLine": common_lib.build_search_line(["data", "google_iam_policy", name, "binding", x, "role"], []),
```
Hi @Tohar-orca, @asofsilva noticed that the line is not correctly detected during the QA checks. Would you mind changing the search lines for the Policies that check the `data` from `resource` to `data`?

Reply:
Of course! Done :)

Reply:
Thanks